fe/fe-core/src/main/java/com/starrocks/common/proc/CompactionsProcDir.java

```diff
@@ -0,0 +1,54 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.common.proc;
+
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.util.TimeUtils;
+import com.starrocks.lake.compaction.CompactionContext;
+import com.starrocks.lake.compaction.CompactionManager;
+import com.starrocks.lake.compaction.PartitionIdentifier;
+import com.starrocks.server.GlobalStateMgr;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class CompactionsProcDir implements ProcDirInterface {
```

Comment:
Better to implement `ProcNodeInterface` if there is no next level and no need to lookup.

Reply:
Fixed, PTAL